tests/hook_main.py

```diff
@@ -3222,3 +3222,14 @@ def create_event_copy(transaction):
         event = EventFactoryBasic()
         db.session.add(event)
         db.session.commit()
+
+
+# ------------------------- Change Password -------------------------
+@hooks.before("Change Password > Reset Forgotten Password > Change Password from Token")
+def change_password_from_token(transaction):
+    """
+    GET /reset-password
+    :param transaction:
```

Comment:
You can do this as well by adding a new user using user factory, and assigning user.reset_password="token"